### PR TITLE
New extension: Beancount Meta

### PR DESCRIPTION
**Pull request by iwfan**
## Description

Quickly generate Beanocunt transactions and save to your journal file. (using [Costflow](https://www.costflow.io/docs/syntax/))

## Screencast

https://user-images.githubusercontent.com/28853200/215390657-fe34284c-35bd-4dac-b0a7-5c286f354845.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/beancount-mate/src/components/ErrorView.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { List } from "@raycast/api";
+
+const ErrorView: React.FC = () => {
+  return (
+    <List.EmptyView
+      icon={"https://cdn-icons-png.flaticon.com/512/4160/4160755.png"}
```
Comment (pernielsentikaer) [inline extensions/beancount-mate/src/components/ErrorView.tsx:7]: Could we save this in the extension instead and in 128x128 size - remember credits to the author/service if needed

File: extensions/beancount-mate/src/components/ErrorView.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      title="Your typings is invalid"
+      description="If you are not familiar with Costflow, Open the syntax Doc ⬇️ "
```
Comment (pernielsentikaer) [inline extensions/beancount-mate/src/components/ErrorView.tsx:9]:
```suggestion
      description="If you are not familiar with Costflow, Open the syntax Doc ⬇️"
```

File: extensions/beancount-mate/src/components/EmptyView.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { List } from "@raycast/api";
+
+const EmptyView: React.FC = () => {
+  return (
+    <List.EmptyView
+      icon={"https://cdn-icons-png.flaticon.com/512/4160/4160766.png"}
```
Comment (pernielsentikaer) [inline extensions/beancount-mate/src/components/EmptyView.tsx:7]: Could we save this in the extension instead and in 128x128 size - remember credits to the author/service if needed

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* Can we handle this when the path is not found

Raycast-Qk7JQx0M@2x

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by iwfan** [comment]
Hi @pernielsentikaer , please do new review for this extension.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Could we resize the smileys to around 128x128 instead? It's quite big right now 😊

Raycast-jeETpf6R@2x

**Comment by pernielsentikaer** [comment]
Hi @iwfan 👋 

I don't think the toast should be here; it makes too much noice.

https://user-images.githubusercontent.com/38033107/219308244-2b24c906-00ee-4163-bb2c-c11741790ba7.mp4

**Comment by iwfan** [comment]
> Hi @iwfan 👋
> 
> I don't think the toast should be here; it makes too much noice.

Hi @pernielsentikaer , You are right, I removed toast when user is typing.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Looks good to me, approved 🔥

**Pull request merged**